bubblewrap.c

```diff
@@ -385,9 +385,11 @@ monitor_child (int event_fd, pid_t child_pid)
           /* We may be getting sigchild from other children too. For instance if
              someone created a child process, and then exec:ed bubblewrap. Ignore them */
           if (died_pid == child_pid)
-            exit (propagate_exit_status (died_status));
+            return propagate_exit_status (died_status);
         }
     }
+
+  return 0; /* Should not be reached... */
```

Comment:
If it shouldn't be reached (it's after a `while (1)` loop with no `break`, as far as I can see), maybe `abort ()` instead?

(GLib code would use `g_assert_not_reached ()` or `g_return_if_reached ()`, but this isn't GLib code.)

Reply:
It's not a bad idea to leave this in. Many compilers will warn that leaving this branch open with no return will lead to undefined behaviour (even though on inspection this is impossible), something like this:

error: no return statement in function returning non-void [-Werror=return-type] 

It prevents a future developer from accidentally leaving undefined behaviour in this function if they change the code above.

Reply:
I didn't suggest removing it, only turning it into an `abort()`. If a future developer accidentally makes that line unreachable, I'd prefer bubblewrap to abort rather than exit successfully.

Reply:
Updated, does it look ok this way?
Thanks